request-manager/request/resumer.go

```diff
@@ -0,0 +1,620 @@
+// Copyright 2018, Square, Inc.
+
+package request
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/rand"
+
+	log "github.com/Sirupsen/logrus"
+	myconn "github.com/go-mysql/conn"
+
+	jr "github.com/square/spincycle/job-runner"
+	"github.com/square/spincycle/proto"
+	"github.com/square/spincycle/request-manager/db"
+)
+
+type Resumer interface {
+	// Suspend marks a running request as suspended and saves the corresponding
+	// suspended job chain.
+	Suspend(requestId string, sjc proto.SuspendedJobChain) error
```

Comment:
If `proto.SuspendedJobChain` has the request ID, why the first arg?

Reply:
no good reason 😬I'll remove it.